===================================================================
RCS file: /home2/webcvs/mirror/kdebindings/smoke/qt/Attic/x_Qt.cpp,v
retrieving revision 1.13.2.2
retrieving revision 1.13.2.3
diff -u -p -r1.13.2.2 -r1.13.2.3
--- kdebindings/smoke/qt/Attic/x_Qt.cpp	2003/03/04 14:33:32	1.13.2.2
+++ kdebindings/smoke/qt/Attic/x_Qt.cpp	2003/09/22 18:07:12	1.13.2.3
@@ -155,7 +155,11 @@ public:
 	x[0].s_enum = (long)Qt::WState_Reserved1;
     }
     static void x_49(Smoke::Stack x) {
-	x[0].s_enum = (long)Qt::WState_Reserved2;
+#if QT_VERSION < 0x030200
+	x[0].s_enum = (long) Qt::WState_Reserved2;
+#else
+	x[0].s_enum = (long) Qt::WState_OwnSizePolicy;
+#endif
     }
     static void x_50(Smoke::Stack x) {
 	x[0].s_enum = (long)Qt::WState_CreatedHidden;
